DEVELOPMENT.md

```diff
@@ -50,4 +50,82 @@ terraform {
 
 provider "intel-cloud" {
   # Configuration options
-}
\ No newline at end of file
+}
+
+## Usage examples
+
+1. Creating a filesystem in IDC
+```shell
+cd examples/resources/filesystems/
+terraform init
+terraform apply
+```
+
+2. Creating a bucket in IDC
+```shell
+cd examples/resources/buckets/
+terraform init
+terraform apply
+```
+
+3. Creating an instance in IDC
+```shell
+cd examples/resources/buckets/
+terraform init
+terraform apply
+```
+
+4. Deploying  gen-ai chatqna
+```shell
+cd examples/gen-ai-xeon-opea-chatqna
+```
+Change the `cloud_init.yaml` file to add your hugging face token (you can create an account on  [Hugging face](https://huggingface.co/settings/tokens) and get a token.
+```shell
+  - echo 'export HUGGINGFACEHUB_API_TOKEN=<HUGGINGFACE_API_TOKEN>' | sudo tee -a /etc/profile.d/opea.sh
+```
+Change the `terraform.tfvars` file to add your `ssh_keyname` , `ssh_pubkey_path` and `ssh_user_email`
+```shell
+idc_region            = "us-region-2"
+ssh_key_name          = "<SSH_KEY>"
+instance_name         = "genai-chatqna-demo3"
+ssh_pubkey_path       = "<PATH_TO_SSH_PUBKEY>"
+ssh_user_email        = "<USER_EMAIL>"
+instance_type         = "vm-large"
+filesystem_name       = "shri-fs6"
+filesystem_size_in_tb = 1
+```
+```shell
+terraform init
+terraform apply
+```
+Verify that instance has been deployed. Follow IDC instructions to ssh into the instance.
```

Comment:
you can remove this step. Once Lucas fixes the upstream repo, it won;t be required.